pkg/reconciler/trigger/trigger_test.go

```diff
@@ -1428,7 +1428,7 @@ func createExchange() *rabbitv1beta1.Exchange {
 		},
 		Spec: rabbitv1beta1.ExchangeSpec{
 			Name:       dlxExchangeName,
-			Vhost:      "/",
```

Comment:
Similar to broker test, there's nothing testing that if a broker's vhost is set, it gets propagated to a trigger's resources.